api/src/main/java/bio/terra/tanagra/service/underlay/ColumnFilter.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.tanagra.service.underlay;
+
+import com.google.auto.value.AutoValue;
+
+/** A filter on a SQL column within a table in an underlay. */
+@AutoValue
+public abstract class ColumnFilter {
+  /** The column to filter on. */
+  public abstract Column column();
```

Comment:
Here's where we define a `Column` property in the Java class for `ColumnFilter`.